maps/clarion.dmm

```diff
@@ -13136,9 +13136,9 @@
 	},
 /area/listeningpost)
 "bLL" = (
-/obj/machinery/door/airlock/syndicate{
-	desc = "This looks like a perfectly innocent weather satellite. It's locked up pretty tight to keep bath salts junkies from breaking in to steal the equipment.";
-	name = "Weather Satellite"
```

Comment:
This name/desc seems like it makes more sense, not sure how other maps are but the hidden syndicate listening post door just being called "Listening Post" seems silly.

Reply:
No other maps have this afaik and the doors I changed are purely internal, I can make a seperate PR changing post door labelling but thats probably best kept seperate